exchange/utils.go

```diff
@@ -17,6 +19,15 @@ import (
 	"github.com/prebid/prebid-server/privacy/lmt"
 )
 
+// cleanMetrics is a struct to export any metrics data resulting from cleanOpenRTBRequests(). It starts with just
+// the TCF version, but made a struct to facilitate future expansion
+type cleanMetrics struct {
+	// A simple flag if GDPR is being enforced on this request.
+	gdprEnforced bool
+	// a zero value means a missing or invalid GDPR string
+	tcfVersion int
```

Comment:
Consider renaming `tcfVersion` to `gdprTcfVersion` to make the groping with `gdprEnforced` obvious as we'll add ccpa, lmt, and coppa in the future.

Reply:
Done